INCORPORATING.md

```diff
@@ -20,13 +20,16 @@ A more complex way to include HIT in your CMake project is to have CMake downloa
 ### 1. Under `third-party/hit` directory, create a file `CMakeLists.txt`, which will be used to download HIT GitHub code.
 ```cmake
 cmake_minimum_required(VERSION 3.12)
-project(AWS_HIT_DOWNLOAD)
+
+project(AWS_HIT_DOWNLOAD 0.1.2) # Change the version number "0.1.2" to whichever version you want
+set(AWS_HIT_BACKEND "lattigo") # This value must be either "lattigo" or "seal"
```

Comment:
This is a PR on the `seal` branch, but this says `lattigo`.

Reply:
This was just sample code in the docs, but that is a valid point. I'll fix it and have the branches be correct.